Core/DefaultVariantManager.swift

```diff
@@ -62,7 +61,9 @@ public struct VariantIOS: Variant {
         VariantIOS(name: "sc", weight: doNotAllocate, isIncluded: When.always, features: []),
         VariantIOS(name: "sd", weight: doNotAllocate, isIncluded: When.always, features: []),
         VariantIOS(name: "se", weight: doNotAllocate, isIncluded: When.always, features: []),
-        VariantIOS(name: "mc", weight: doNotAllocate, isIncluded: When.inEnglish, features: [.newSuggestionLogic]),
```

Comment:
Clean up the previous history experiment but leave existing users allocated.  Once we get to 100% rollout we can remove the `md` variant.